pkg/dispers/dispers/requests.go

```diff
@@ -3,6 +3,7 @@ package dispers
 import (
 	"encoding/json"
 	"errors"
+  "time"
```

Comment:
An identation is missing

Reply:
Next time I resolve conflicts from Atom !

Reply:
I'm curious of how you resolve conflicts if not in your IDE?

Reply:
GitHub in Browser. Pretty simple but enough to do copy/paste in this case (except for indents..).